jetty-util/src/test/java/org/eclipse/jetty/util/URIUtilCanonicalPathTest.java

```diff
@@ -114,6 +114,13 @@
                 // paths with encoded segments should remain encoded
                 // canonicalPath() is not responsible for decoding characters
                 {"%2e%2e/", "%2e%2e/"},
+                {"/%2e%2e/", "/%2e%2e/"},
+
+                // paths with parameters are not elided
+                // canonicalPath() is not responsible for decoding characters
+                {"/foo/.;/bar", "/foo/.;/bar"},
+                {"/foo/..;/bar", "/foo/..;/bar"},
+                {"/foo/..;/..;/bar", "/foo/..;/..;/bar"},
```

Comment:
This is an unrelated change.

Reply:
Yes, see note explaining it above.

Reply:
Oh no it's another unrelated change!   Double ooops

Reply:
These are useful and good changes.
Just would have liked to see them in a different commit explaining why they were made.
Right now they seem to be related to the change for commit 36b2a4bbc9e628185747d32c4044067bb2b15419 (Issue #5088 Review ContextHandler Locking), which will be super confusing in the future.

Reply:
yep - my bad!